js/views/message_list_view.js

```diff
@@ -17,31 +17,29 @@
             if (this.$el.scrollTop() === 0) {
                 this.$el.trigger('loadMore');
             }
-            if (this.bottomOffset === 0) {
+            if (this.atBottom()) {
                 this.$el.trigger('atBottom');
             } else if (this.bottomOffset > this.outerHeight) {
                 this.$el.trigger('farFromBottom');
             }
         },
+        atBottom: function() {
+            return this.bottomOffset < 20;
+        },
         measureScrollPosition: function() {
             if (this.el.scrollHeight === 0) { // hidden
                 return;
             }
             this.outerHeight = this.$el.outerHeight();
             this.scrollPosition = this.$el.scrollTop() + this.outerHeight;
             this.scrollHeight = this.el.scrollHeight;
-            this.shouldStickToBottom = this.scrollPosition === this.scrollHeight;
-            if (this.shouldStickToBottom) {
-                this.bottomOffset = 0;
-            } else {
-                this.bottomOffset = this.scrollHeight - this.$el.scrollTop() - this.outerHeight;
-            }
+            this.bottomOffset = this.scrollHeight - this.$el.scrollTop() - this.outerHeight;
```

Comment:
We don't use `shouldStickToBottom` anywhere, so we're good there.

Going further, if we expand out the meaning of `this.scrollPosition`, the removed line 33 becomes:

```
this.shouldStickToBottom = (this.$el.scrollTop() + this.outerHeight) === this.scrollHeight;
```

Which means that the old and new code ultimately result in the same values for `this.bottomOffset`:

```
// zero if the first value equals the second two values added together
this.bottomOffset = this.scrollHeight - (this.$el.scrollTop() + this.outerHeight);
```